backend/mora/service/manager.py

```diff
@@ -13,20 +13,145 @@
 
 """
 import uuid
+import flask
 
 from . import address
 from . import handlers
+from . import orgunit
+from . import facet
+from . import employee
 from .. import common
 from .. import lora
 from .. import mapping
 from .. import util
 from .. import validator
 
+SEARCH_FIELDS = {
+    'e': 'tilknyttedebrugere',
+    'ou': 'tilknyttedeenheder'
+}
 
-class ManagerRequestHandler(handlers.OrgFunkRequestHandler):
+
+class ManagerRequestHandler(
+    handlers.OrgFunkRequestHandler,
+    handlers.ReadingRequestHandler
+):
+
+    __slots__ = ()
     role_type = 'manager'
     function_key = mapping.MANAGER_KEY
 
+    @classmethod
+    def has(cls, scope, objid):
+        pass
+
+    @classmethod
+    def get(cls, c, type, objid):
+
```

Comment:
Considering that this is almost entirely the same as the method in `address.py`, I think we should move it to a shared superclass in `mapping.py` called `OrgFunkReadingRequestHandler`.

Reply:
Jeg har specifikt aftalt med Christoffer at vi kopierer den i første omgang. De er faktisk heller ikke ens.